docs/docs/recipes/integrate-logto/android.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_label: Android (Kotlin)
+sidebar_label: Android
```

Comment:
maybe better keep ` (Kotlin)`?

Reply:
Why? Java developers can also use this documentation.